assets/styles/main.scss

```diff
@@ -808,14 +811,19 @@ body.sticky-offset {
         vertical-align: top;
 
         &.prop-name {
-            width: 300px;
+            @include desktop {
+                width: 300px;
+            }
         }
         td.prop-name {
             width: auto;
         }
         &.item-action {
             width: 35px;
         }
+        @include mobile {
+            display: block;
```

Comment:
You don't want to apply that to any `td` (`.prop-value` `td` contains another `table` with its own `td`s, which causes the messed up styling). Instead, you want to scope this to `td.prop-value` and `td.prop-name`:

![image](https://user-images.githubusercontent.com/4048809/135717134-c8dc1c6e-59ad-4b07-9f95-00a6edcf521c.png)

Reply:
Thanks!